sample/jvm/src/main/java/co/yml/ychat/jvm/controller/YChatController.java

```diff
@@ -32,6 +32,14 @@ public ResponseEntity<String> chatCompletions(
         return ResponseEntity.ok(result);
     }
 
+    @GetMapping("generations")
```

Comment:
I recommend updating the README file of the JVM sample to include instructions on how to use this endpoint. You can refer to the description provided in the previous endpoint sample as a guide.

What do you think?

Reply:
Done!